server/argv.js

```diff
@@ -0,0 +1,17 @@
+module.exports = require('yargs')
+    .options({
+        c: {
+            alias: 'config',
+            default: 'configs/config.json',
+            describe: 'Configuration file path',
+            nargs: 1,
+            type: 'string'
+        },
+        s: {
+            alias: 'socket',
+            describe: 'Socket the app will listen to',
```

Comment:
Pair of port number and IP address (which is implicitly is `0.0.0.0`) is a socket. Actually, to be pedantic, it's not, but file path is not either.